packages/editor/src/components/post-panel-row/style.scss

```diff
@@ -1,16 +1,14 @@
 .editor-post-panel__row {
 	width: 100%;
 	min-height: $grid-unit-40;
-	justify-content: flex-start !important;
-	align-items: flex-start !important;
 }
 
 .editor-post-panel__row-label {
 	width: 38%;
 	flex-shrink: 0;
 	min-height: $grid-unit-40;
 	display: flex;
-	align-items: center;
+	align-items: flex-start;
```

Comment:
Since [the parent now has a new flex-start style](https://github.com/WordPress/gutenberg/pull/70981/files), is it safe to remove this style?